src/GovUk.Education.ExploreEducationStatistics.Admin/Services/FileUploadsValidatorService.cs

```diff
@@ -74,6 +74,11 @@ public class FileUploadsValidatorService : IFileUploadsValidatorService
             {
                 return ValidationActionResult(FileCannotBeEmpty);
             }
+            
+            if (file.Length > 2147483648)
```

Comment:
Don't know if it's perhaps worth sticking this in as a constant to make its choice a bit more obvious perhaps?  Like `TWO_GB" or similar?